SimCalorimetry/EcalSimProducers/python/ecalSimParameterMap_cff.py

```diff
@@ -10,7 +10,17 @@
     simHitToPhotoelectronsBarrel = cms.double(2250.0),
     syncPhase = cms.bool(True),
     doPhotostatistics = cms.bool(True),
-    photoelectronsToAnalogBarrel = cms.double(0.000444444),
-    timeDependent = cms.bool(False)
+    photoelectronsToAnalogBarrel = cms.double(0.000444444)
+)
+
+ecal_sim_parameter_map_ph2 = cms.PSet(
+    readoutFrameSize = cms.int32(16),
```

Comment:
ecalPh2::sampleSize is a constexpr in EcalConstants.h. Could it be used instead of this parameter?